<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortierAddVisit</class>
 <widget class="QDialog" name="PortierAddVisit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="return_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Powrót</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>871</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>411</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="check_assignment">
      <property name="text">
       <string>Sprawdź zapis</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="check_aviability">
      <property name="text">
       <string>Sprawdź dostępność</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_visit">
      <property name="text">
       <string>Dodaj wejście</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>10</y>
     <width>101</width>
     <height>47</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="date">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2018</year>
        <month>6</month>
        <day>25</day>
       </datetime>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2018</year>
        <month>6</month>
        <day>25</day>
       </datetime>
      </property>
      <property name="currentSectionIndex">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>251</width>
     <height>55</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="student_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Student</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="guest_button">
        <property name="text">
         <string>Gość</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>PESEL Studenta</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pesel_input"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
